app/renderer.js

```diff
@@ -14,34 +16,80 @@ const $recipeContainer = $('.recipe-list-container');
 const $saveButton = $('.save-recipe-button');
 const $searchButton = $('.search-button');
 const $searchInput = $('.search-input');
+const $deleteButton = $('.delete-button');
 const $seeAllButton = $('.see-all-button');
 const $servings = $('#servings');
 const $time = $('#time');
 const $newDirections = $('.new-direction-button');
+const array = [];
```

Comment:
We will probably want to change this variable name on the next iteration as it feels a little yucky.